tests/src/action.spec.ts

```diff
@@ -184,6 +184,7 @@ describe('src/action', () => {
 					pullRequestId: 4423422,
 				},
 			});
+			expect(createTagStub).to.have.not.been.called;
```

Comment:
If you want to create a tag on finalize then the draft release will inherently also have createTag: true. Therefore, with this check in place we should modify the input to also include `createTag: true` to ensure on draft release a tag is not created.